app/forms/move_location_form.rb

```diff
@@ -61,7 +61,10 @@ def check_user
   end
 
   def check_parent_location
-    return if parent_location.present?
+    if parent_location.present?
+      errors.add(:parent_location, "cannot be a building") if parent_location.location_type.name == "Building"
```

Comment:
I think this might be better to check if it is a container (something that can store labware) because this would cover building, room, car park etc.

Reply:
👍